src/main/resources/application-local.yml

```diff
@@ -0,0 +1,23 @@
+server:
+  port: 5040
+
+spring:
+  rabbitmq:
+    addresses: localhost
+    port: 5672
```

Comment:
To remove ?

Reply:
I am not sure, should run a test in local, but I see this property is declared in powsybl-ws-commons

Reply:
Yes, we get it from powsybl-ws-commons

Reply:
we can remove only port: 5672, addresses is used for local test

Reply:
DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c